### PR TITLE
Removing Configuration

### PR DESCRIPTION
**Pull request by ntfshard**
Second edition, expanded, improved

Tested Alica Plan import for:
- ./supplementary/alica_ros2/alica_ros2_turtlesim/etc
- ./supplementary/alica_ros1/supplementary_tests/src/test/etc
- ./supplementary/alica_ros1/alica_ros_turtlesim/etc
- ./supplementary/constraintsolver/test/etc
- ./alica_tests/src/test/etc

### CODE REVIEW

**Review by Dale-Koenig — APPROVED** [review]

**Comment by rogeriofonteles** [comment]
I don't know if necessary, but I saw we also have `rr_sootballs/lbc_bridge/sootballs_alica_base/etc`, idk if this has to work for this also.

**Comment by ntfshard** [comment]
> I don't know if necessary, but I saw we also have `rr_sootballs/lbc_bridge/sootballs_alica_base/etc`, idk if this has to work for this also.

Seems imports fine

File: alica_tests/src/test/etc/plans/ConfigurationTestPlan.pml [hunk 1]
```diff
@@ -40,52 +40,7 @@
   "states": [
     {
       "comment": "",
-      "confAbstractPlanWrappers": [
```
Comment (veerajsk) [inline alica_tests/src/test/etc/plans/ConfigurationTestPlan.pml:43]: can delete this file right?

Comment (ntfshard) [reply alica_tests/src/test/etc/plans/ConfigurationTestPlan.pml:43]: File?

Comment (veerajsk) [reply alica_tests/src/test/etc/plans/ConfigurationTestPlan.pml:43]: yes, since Configurations are gone, we don't need any tests related to it

**Review by Dale-Koenig — COMMENTED** [review]
Can remove 
static const std::string configuration = "configuration";
static const std::string configuration_extension = ".cfg";

from Strings.h

**Review by Dale-Koenig — APPROVED** [review]

**Review by veerajsk — APPROVED** [review]

**Pull request merged**